runner/local/simple_test.go

```diff
@@ -112,6 +114,10 @@ func aborted() runner.ProcessStatus {
 	return runner.AbortStatus(runner.RunId(""))
 }
 
+func badRequest(errorText string) runner.ProcessStatus {
+	return runner.BadRequestStatus(runner.RunId(""), fmt.Errorf(errorText))
+}
+
 func assertRun(t *testing.T, r runner.Runner, expected runner.ProcessStatus, args ...string) runner.RunId {
```

Comment:
Ooh; I like that pattern.

FWIW, since writing this code I've heard that you're supposed to only t.Fatal on the outermost level.... so instead of assert, we should test and then if err != nil t.Fatal... but that's so much ode.

Would it be nicer to include the full stack trace? That's at least populated automatically?